scipy/integrate/src/lsoda.h

```diff
@@ -0,0 +1,85 @@
+#ifndef __LSODA_H
+#define __LSODA_H
```

Comment:
Names with two leading underscores (and names with a single underscore followed by a capital letter) are reserved for the compiler and should not be used in user code. Remove the underscores, and maybe follow a guideline that I've seen before (google's?) to use more of the path in the include guard, e.g `INTEGRATE_SRC_LSODA_H`.  Given the context, though, `LSODA_H` is probably fine, too.